frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -189,7 +182,7 @@ export const AboutResourcePage = ({
    *
    * @param s the selected string
    */
-  const onChangeResourceType = (s: string) => {
+  const handleChangeResourceType = (s: string) => {
     if (s === 'Standard') setResourceType('Default');
     else if (s === 'System ressurs') setResourceType('Systemresource');
     else if (s === 'Maskinporten skjema') setResourceType('MaskinportenSchema');
```

Comment:
we can write this method more readable and more performant. Suggest something more like this:
```suggestion
  const handleChangeResourceType = (type: string) => {
    const resourceTypeMap = {
      Standard: "Default",
      "System ressurs": "Systemresource",
      "Maskinporten skjema": "MaskinportenSchema",
    };
    setResourceType(resourceTypeMap[type]);
    setHasResourceTypeError(!Object.values(resourceTypeMap).includes(type));
  };
```

Then we do not need to repeat Standard, System Resurs and Maskinporten skjema multiple times. Lookup within a map is also more performant than checking all the if and else.

Reply:
When implementing translations I think we should create an enum/map that makes the keys static regardless of the language itself. :)

Reply:
Thanks, this is really nice! :D